src/modality-specific-files/magnetoencephalography.md

```diff
@@ -26,7 +26,7 @@ and a guide for using macros can be found at
 {{ MACROS___make_filename_template(
    "raw",
    datatypes=["meg"],
-   suffixes=["meg", "markers", "events", "physio", "stim"])
+   suffixes=["meg", "markers", "events", "physio", "stim", "digitizer"])
```

Comment:
perhaps `_digitizer` should be removed from the spec entirely:

- #1560

Reply:
removed